RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -458,7 +406,7 @@ boolean auto_post_adventure()
 
 	if(auto_have_skill($skill[Summon Smithsness]) && (my_mp() > 20))
 	{
-		use_skill(1, $skill[Summon Smithsness]);
+		useTomeSkill(1, $skill[Summon Smithsness]);
 	}
```

Comment:
This and below replace calls to use_skill with the new useTomeSkill and useLibramSkill that I added to allow accessing smithsness items in G-Lover and any other path in the future that for some reason can use the bookshelf but can't use the skills themselves.